lib/shelley/src/Cardano/Wallet/Shelley/Network/Blockfrost.hs

```diff
@@ -884,6 +881,8 @@ fromBlockfrostPP network BF.ProtocolParams{..} = do
                     }
         , maximumCollateralInputCount = maxCollateralInputs
         , minimumCollateralPercentage = collateralPercent
+        -- TODO: Surely this is not right:
+        , minimumUTxO = MinimumUTxONone
```

Comment:
I've added a task to the PR description to determine the correct way to initialize this field.

Reply:
Update: in a discussion with @Unisay, we decided to make this PR preserve the existing behaviour, so that the Blockfrost network layer continues to specify the Alonzo-specific coins-per-UTxO-word-based function. The only effect of applying this PR should be that the choice will become more explicit and obvious.